packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -19,16 +19,28 @@ export enum AutoUpdaterEvents {
 export const defaultUpdateInterval = 12 * 60 * 60 * 1000;
 export const initialUpdateStartup = 5 * 1000;
 const releasesLink = 'https://objects.joplinusercontent.com/r/releases';
-const supportedPlatformAssets: { [key in string]: string } = {
-	'darwin': 'latest-mac.yml',
-	'win32': 'latest.yml',
+
+interface PlatformAssets {
+	[platform: string]: {
+		[arch: string]: string;
```

Comment:
Please define a type `export type Architecture = typeof process.arch;` and use this for the architecture type everywhere.